### DISCUSSION TITLE
Box art different

### DISCUSSION THREAD

**Discussion by GhermanX** [OnionUI/Onion#573] [Ideas]
Is it possible to have two images for the roms? 
One on the choice screen and a different one on the GAMEINFO screen.

it would be interesting to have a cover image on one side and a screenshoot of the game at gameinfo

**Comment by Olywa** [comment]
It's a nice idea but is not currently possible unfortunately. 

Miyoo have not released the source code for the binaries, which would be needed to deviate from the one-image-per-rom artwork behaviour implemented in stock.
